js/gemojify.js

```diff
@@ -2,6 +2,7 @@ function gemojify() {
   var elementsToReplace = document.getElementsByClassName('gemojify');
   for (var i = 0; i < elementsToReplace.length; i++) {
     elementsToReplace[i].innerHTML = elementsToReplace[i].innerHTML
+      .replace(':+1:', ':_1:') // special character not compatible with CSS
```

Comment:
This will only replace one occurrence
```
':+1::+1:'.replace(':+1:',':_1:')
> ':_1::+1:'
```
Turn it into a regex like the one below? Change `':+1:'` to `/:\+1:(?![^<]*>)/g` to only replace things outside `<>` tags